playground/compiler-express/package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "build": "react-router build",
     "dev": "node ./server.js",
+    "setup": "pnpm i react-router@workspace",
```

Comment:
Feels like this should be handled for us by `pnpm`. Is our `pnpm` workspace messed up or is this a limitation of how we programmatically create playgrounds?

Reply:
I'm not sure - I agree the binary should be properly installed on `pnpm i` - and it is in an external reproduciton not using `workspace:*` symlinks.  So I think it's either an issue with our setup, or an issue with pnpm and `workspace:*` deps?